packages/amplication-client/src/Resource/git/AuthResourceWithBitbucketCallback.tsx

```diff
@@ -1,14 +1,16 @@
 import { useEffect } from "react";
-import { useMutation } from "@apollo/client";
 import { useTracking } from "../../util/analytics";
 import { AnalyticsEventNames } from "../../util/analytics-events.types";
-import { EnumGitProvider } from "../../models";
-import { COMPLETE_OAUTH2_FLOW } from "./queries/git-callback";
+import {
+  EnumGitProvider,
+  useCompleteGitOAuth2FlowMutation,
+} from "../../models";
 import { GET_PROJECTS } from "../../Workspaces/queries/projectQueries";
 
 const AuthResourceWithBitbucketCallback = () => {
   const { trackEvent } = useTracking();
-  const [completeAuthWithGit] = useMutation<boolean>(COMPLETE_OAUTH2_FLOW, {
+
+  const [completeAuthWithGit] = useCompleteGitOAuth2FlowMutation({
```

Comment:
Is this related to what you did with graphql codegen?

Reply:
yep, a pure 1-2-1 replacement